patches/minecraft/net/minecraft/entity/player/InventoryPlayer.java.patch

```diff
@@ -1,5 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/entity/player/InventoryPlayer.java
 +++ ../src-work/minecraft/net/minecraft/entity/player/InventoryPlayer.java
+@@ -18,7 +18,7 @@
+ import net.minecraftforge.fml.relauncher.Side;
+ import net.minecraftforge.fml.relauncher.SideOnly;
+ 
+-public class InventoryPlayer implements IInventory
++public class InventoryPlayer implements IInventory, net.minecraftforge.items.IItemHandlerModifiable
+ {
```

Comment:
Where is this queried? The point of Capabilities is that we don't have to implement them in the static class.
Tho I like the concept of doing a standard way to add extra slots to the player because everyone and their dog seems to be doing this... I think this is a seperate issue.
A small anon class or something in EntityPlayer would probably be the better option for this.